iib/workers/config.py

```diff
@@ -10,31 +12,31 @@ class Config(object):
     """The base IIB Celery configuration."""
 
     # When publishing a message, don't continuously retry or else the HTTP connection times out
-    broker_transport_options = {'max_retries': 10}
+    broker_transport_options: Dict[str, Any] = {'max_retries': 10}
     # Avoid infinite Celery retries when the broker is offline.
     broker_connection_max_retries: int = 10
-    iib_aws_s3_bucket_name = None
-    iib_api_timeout = 30
+    iib_aws_s3_bucket_name: Optional[str] = None
+    iib_api_timeout: int = 30
     iib_docker_config_template = os.path.join(
         os.path.expanduser('~'), '.docker', 'config.json.template'
     )
-    iib_greenwave_url = None
+    iib_greenwave_url: Optional[str] = None
     iib_grpc_init_wait_time = 30
     iib_grpc_max_port_tries = 100
     iib_grpc_max_tries = 5
     iib_grpc_start_port = 50051
     iib_image_push_template = '{registry}/iib-build:{request_id}'
```

Comment:
I think these values are missing the static type as well as some other config values

Reply:
I just added static types to all class variables (most of them in config.py). 
I intended to add static types just to method definitions at first, that´s why it was without them. 
Is it okay now?

Reply:
Yes, thanks!